### ISSUE TITLE
Uncaught Error: getaddrinfo ENOTFOUND avatars2.githubusercontent.com

### ISSUE THREAD

**Issue by miguel4ngelc** [atom/atom#5811]
I'm

1. ... Trying to install packages
2. ... Search and press Install
3. ... Error appears

**Atom Version**: 0.170.0
**System**: linux 3.13.0-24-generic
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: getaddrinfo ENOTFOUND avatars2.githubusercontent.com

```
At stream.js:94

Error: getaddrinfo ENOTFOUND avatars2.githubusercontent.com
  at errnoException (dns.js:41:10)
  at Object.onlookup [as oncomplete] (dns.js:91:26)

```

### Commands

```
     -4:59.5 core:backspace (atom-text-editor.editor.mini)
     -4:58.0 core:confirm (atom-text-editor.editor.mini)
     -0:02.8 core:paste (atom-text-editor.editor.mini)
     -0:00.0 core:confirm (atom-text-editor.editor.mini)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "language-objective-c"
    ],
    "themes": [
      "seti-ui",
      "seti-syntax"
    ]
  },
  "editor": {
    "fontFamily": "Source Code Pro",
    "lineHeight": 1.5,
    "invisibles": {}
  }
}
```

### Installed Packages

```coffee
# User
seti-syntax, v0.2.1
seti-ui, v0.6.1
vim-mode, v0.34.0

# Dev
No dev packages
```

**Comment by benogle** [comment]
cc @thedaniel how might settings view trigger this? I know that `navigator.onLine` is always true for linux.

**Comment by thedaniel** [comment]
IIRC settings-view uses `navigator.onLine` to decide whether to hit the avatar cache or not, but is intended to recover from a failure to load avatars with a placeholder regardless of `navigator.onLine`'s state. if it doesn't, that's definitely a bug to be filed on settings-view.

**Comment by kevinsawicki** [comment]
This is a duplicate of https://github.com/atom/atom/issues/4986

It is fixed in the latest release of Atom, 0.184
